wazp/callbacks/metadata.py

```diff
@@ -1,114 +1,165 @@
-import base64
 import pathlib as pl
 
 # import pdb
 import re
-from typing import Any
 
 import dash
 import dash_bootstrap_components as dbc
+import pandas as pd
 import utils
-import yaml
 from dash import Input, Output, State, dash_table, html
 
-VIDEO_TYPES = [".avi", ".mp4"]
 # TODO: other video extensions? have this in project config file instead?
+VIDEO_TYPES = [".avi", ".mp4"]
 
 
-def get_home_callbacks(app: dash.Dash) -> None:
-    """Return all callback functions for the home tab.
+##########################
+# Fns to create components
+###########################
+def create_metadata_table_component_from_df(
```

Comment:
I like that you moved this function to the `metadata.py`. Makes more sense, than in `utils.py`